sys/cheri/c18n.h

```diff
@@ -85,6 +85,19 @@ struct cheri_c18n_info {
 	void * __kerncap	comparts;
 };
 
+/*
+ * The interface provided by the kernel via sysctl for compartmentalization
+ * monitoring tools such as procstat.
+ */
+#define	CHERI_C18N_COMPART_MAXNAME	56
```

Comment:
I worry this should be more like PATH_MAX + NAME_MAX + 2 (separator and terminator).